react/__tests__/helpers/intlHelper.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types'
+import { IntlProvider, intlShape } from 'react-intl'
+
+export const DEFAULT_LOCALE = 'en-US'
+
+export function getIntlContextInfo() {
+  const context = {
+    culture: {
+      currency: 'USD',
+    },
+  }
+
+  const childContextTypes = {
+    intl: intlShape,
+    culture: PropTypes.object,
+  }
+
+  return { context, childContextTypes, locale: DEFAULT_LOCALE }
+}
+
+export function getIntlInstance(props = { locale: DEFAULT_LOCALE }, context = {}) {
+  props.messages = props.messages || getMessages(props.locale)
```

Comment:
Don't alter function parameters. Create a new variable or use the object destructuring on usage, e.g
```js
const intlProvider = new IntlProvider({ message: props.message || getMessages(props.locale), ...props }, context)
```